application/src/main/java/org/togetherjava/tjbot/commands/basic/VcActivityCommand.java

```diff
@@ -223,51 +214,29 @@ private static void handleErrors(@NotNull SlashCommandEvent event,
         logger.warn("Something went wrong in the VcActivityCommand", throwable);
     }
 
-
     /**
-     * This grabs the OptionMapping, after this it <br />
-     * - validates whenever it's within an {@link Integer Integer's} range <br />
-     * - validates whenever it's positive <br />
+     * This grabs the OptionMapping, after this it returns null of the OptionMapping is null, else
+     * it'll return the number option as an Integer
      *
      * <p>
      * <p/>
      *
-     * @param event the {@link SlashCommandEvent}
      * @param optionMapping the {@link OptionMapping}
      * @return nullable {@link Integer}
-     * @throws java.lang.IllegalArgumentException if the option's value is - outside of
-     *         {@link Integer#MAX_VALUE} - negative
-     */
-    @Contract("_, null -> null")
-    private static @Nullable Integer handleIntegerTypeOption(@NotNull SlashCommandEvent event,
+     **/
+    @Contract("null -> null")
+    private static @Nullable Integer handleIntegerTypeOption(
```

Comment:
With all the error handling removed, this method is obsolete. Inline the logic and remove the method.

Reply:
Or at least rename it, its not a "handle" method anymore. `requireIntOptionIfPresent` and clean up the code
```java
return option == null ? null : Math.toIntExact(option.getAsLong());
```
and also the javadoc then, its written a bit flopsy.

Reply:
Thank you for the guidance, resolved it.